irods_testing_environment/irods_config.py

```diff
@@ -387,3 +389,65 @@ def configure_irods_federation_testing(ctx, remote_zone, zone_where_tests_will_r
     if execute.execute_command(container, asq, user='irods') is not 0:
         raise RuntimeError('failed to create specific query [{}] [{}]'
                            .format(bug_3466_query, container.name))
+
+
+def configure_pam_for_auth_plugin(docker_client, compose_project):
+    """Add lines required for PAM legacy/pam_password auth plugin to work across all platforms.
+
+    Arguments:
+    docker_client -- docker client for interacting with the docker-compose project
+    compose_project -- compose.Project in which the iRODS servers are running
+    """
+    from . import archive
+
+    import concurrent.futures
+    import textwrap
+
+    def configure_pam(docker_client, docker_compose_container, path_to_config, contents):
+        container = docker_client.containers.get(docker_compose_container.name)
+
+        archive.put_string_to_file(container, path_to_config, contents)
+
+        # TODO: run /usr/sbin/irodsPamAuthCheck here to make sure it's okay
```

Comment:
Do we want to capture this in an issue?

Reply:
Done

https://github.com/irods/irods_testing_environment/issues/133